services/dashboard/views/apps/apps.js

```diff
@@ -2,64 +2,78 @@
  * Licensed under the Apache License, Version 2.0
```

Comment:
The current boundary of scalajs integration is about the model layer. `apps.js` is about how model renders into view. I'd perfer we do the refactoring stage by stage, if it is possible.